src/tests.ts

```diff
@@ -54,6 +54,8 @@ userSchema.plugin(passportLocalMongoose, {
   attemptsField: "attempts",
   maxAttempts: 3,
   usernameCaseInsensitive: true,
+  interval: process.env.NODE_ENV === "test" ? 1 : 100,
+  maxInterval: process.env.NODE_ENV === "test" ? 1 : 300000,
```

Comment:
@kjkurtz Just for my own learning, how come we need this ternary here?

Reply:
If we are running tests, we want it to not temporarily lock the user account otherwise tests will fail since they run really fast. It gets set to 1ms which allows the tests to pass then (I'd have done 0, but then it sees it as not set/undefined and it goes to the default which is too high). But in the real world, we can introduce a required delay between sign-in attempts.